docs/agents.md

```diff
@@ -669,6 +669,51 @@ except UsageLimitExceeded as e:
     - Usage limits are especially relevant if you've registered many tools. Use `request_limit` to bound the number of model turns, and `tool_calls_limit` to cap the number of successful tool executions within a run.
     - The `tool_calls_limit` is checked before executing tool calls. If the model returns parallel tool calls that would exceed the limit, no tools will be executed.
 
+##### Soft Tool Call Limits with `max_tool_calls`
```

Comment:
I think this belongs on `tools-advanced.md`, with a link from the usage limits section above.

And no need to mention the name of the field in the title, that'll make it too long for the ToC sidebar